### PR TITLE
Changes for cocoapods/cocoapods#7908

### PR DESCRIPTION
**Pull request by jkap**
xcconfigs got renamed (as expected) so this should fix that

### CODE REVIEW

**Comment by jkap** [comment]
@dnkoutso @segiddins

**Comment by dnkoutso** [comment]
For https://github.com/CocoaPods/CocoaPods/pull/7908

**Comment by dnkoutso** [comment]
@jkap this needs a rebase it included more changes that needed for this PR.

**Comment by jkap** [comment]
@dnkoutso if this looks good to you now I'd like to merge this so that I can have a final sha for CocoaPods/CocoaPods#7908

File: install_static_swift_modules/after/Pods/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap [hunk 1]
```diff
@@ -1 +1 @@
-/Users/segiddins/Development/Rainforest/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.modulemap
\ No newline at end of file
+/Users/jkap/Development/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.modulemap
```
Comment (amorde) [inline install_static_swift_modules/after/Pods/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap]: re: discussion on Slack, these don't need to change but eventually we should build something that makes these work independently of the developer's file system structure

Comment (jkap) [reply install_static_swift_modules/after/Pods/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap]: wait shit i thought i got all of these. hold on a sec

**Pull request merged**